decidim-core/lib/decidim/coauthorable.rb

```diff
@@ -82,6 +82,14 @@ def creator
         coauthorships.order(:created_at).first
       end
 
+      # Checks whether the user is the creator of the given proposal
+      #
+      # author - the author to check for authorship
+      def created_by?(author)
+        return false if author.nil?
```

Comment:
This is unnecessary if you apply the following suggestion

Reply:
Thanks